core/src/main/scala/org/broadinstitute/dsde/rawls/workspace/MultiCloudWorkspaceService.scala

```diff
@@ -269,6 +278,10 @@ class MultiCloudWorkspaceService(override val ctx: RawlsRequestContext,
             workspaceManagerDAO.enableApplication(workspaceId, wsmConfig.leonardoWsmApplicationId, context)
           })
         }
+
+        // create a WDS application in Leo
+        _ <- createWdsAppInWorkspace(workspaceId, parentContext)
```

Comment:
Just checking - this is expected to fail until cloning is fully developed?  I tried cloning a workspace on my bee and this call went through - it attempted to create a CROMWELL app - but failed due to lack of a storage container.

Reply:
ahh, good catch. It is not expected to fail. I moved WDS(+CBAS) app creation after storage container creation for clones - but I haven't retested yet to validate that, I will do that as soon as I can

Reply:
@calypsomatic update: this is now working for clones, in my testing.